cmd/cluster-network-operator/leader.go

```diff
@@ -29,7 +27,7 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
-var log = logf.Log.WithName("leader")
+var logger2 = logf.Log.WithName("leader")
```

Comment:
Just call this `logger` and use it for everything.

Reply:
ack